client/src/api/axios.ts

```diff
@@ -14,4 +14,43 @@ const BaseClient: AxiosInstance = axios.create({
   timeout: 1000,
 });
 
-export { AuthClient, BaseClient };
+
+async function GetPlans(projectId:any){
+  await AuthClient.get(`/test_plan/${projectId}/`)
+  .then(response=>{
+    return response.data
+  })
+  .catch(error=>{
+    return error
+  })
+}
+
+async function CreateNewTestSuite(testSuiteDetails: any, project_id:any){
+  await AuthClient.post(`/test_suites/${project_id}/`)
+  .then(response=>{
+  })
+  .catch(error=>{
+    return error
+  })

```

Comment:
```suggestion
  return await AuthClient.post(`/test_suites/${project_id}/`)
 
```
handle it with try and catch when you call it